examples/src/main/scala/ai/h2o/sparkling/examples/AirlinesWithWeatherDemo2.scala

```diff
@@ -1,175 +0,0 @@
-/*
-* Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
This is almost identical to AirlinesWithWeatherDemo except it runs GBM at the end. Added GBM to AirlinesWithWeatherDemo to avoid duplicated tests